tests/conftest.py

```diff
@@ -84,6 +87,40 @@ def db(tmpdir, make_db):
     return make_db(path)
 
 
+def prepare_account(addr, mailcow, db_path):
+    password = mailcow.auth["X-API-Key"]
+    mailcow.add_user_mailcow(addr, password, "admbot")
+    ac = deltachat.Account(str(db_path))
```

Comment:
Here you can add

```
 ac._evtracker = ac.add_account_plugin(FFIEventTracker(ac))
``` 

after importing FFIEventTracker from deltachat.events

Reply:
In any case it makes sense to turn on logging here for debugging purposes :) good idea